### PR TITLE
[Merged by Bors] - Document how `Style`'s size constraints interact

### PR DESCRIPTION
**Pull request by ickshonpe**
# Objective

Document how `Style`'s size constraints interact.

# Changelog
* Added extra doc comments to `Style`'s size fields.

### CODE REVIEW

**Review by alice-i-cecile — APPROVED** [review]

**Comment by mockersf** [comment]
I'm not sure what you mean by the word "respective" in each doc string?

**Comment by ickshonpe** [comment]
Respective size value meaning "Width or Height", it is a bit awkward.

**Comment by ickshonpe** [comment]
Writing them both out explicitly is clearest:
```
/// The `width` of `size` is used when it is within the bounds defined by `min_size.width` and `max_size.width`.
/// The `height` of `size` is used when it is within the bounds defined by `min_size.height` and `max_size.height`.
```
but it feels a bit silly.

**Comment by mockersf** [comment]
> Respective size value meaning "Width or Height", it is a bit awkward.

Oh that was really not what I was understanding. For me the value is the value of the whole struct, not only of one of its fields.

File: crates/bevy_ui/src/ui_node.rs [hunk 1]
```diff
@@ -248,11 +248,20 @@ pub struct Style {
...
     pub size: Size,
     /// The minimum size of the flexbox
+    ///
+    /// `min_size.width` is used if it is greater than either `size.width` or `max_size.width`, or both.
+    /// `min_size.height` is used if it is greater than either `size.height` or `max_size.height`, or both.
     pub min_size: Size,
     /// The maximum size of the flexbox
+    ///
+    /// `max_size.width` is used if it is within the bounds defined by `min_size.width` and `size.width`.
+    /// `max_size.height` is used if it is within the bounds defined by `min_size.height` and `size.height.
```
Comment (mockersf) [inline crates/bevy_ui/src/ui_node.rs:264]: shouldn't that be "smaller" or "lesser" rather than "within the bounds"?

**Review by mockersf — APPROVED** [review]

**Review by alice-i-cecile — APPROVED** [review]
bors r+

**Pull request closed without merging**